cpp/data/installer/scripts/install_centos_packages.sh

```diff
@@ -35,6 +35,7 @@ dnf --assumeyes install curl epel-release wget
 dnf config-manager --set-enabled PowerTools
 dnf config-manager --add-repo https://raw.githubusercontent.com/ubtue/ub_tools/master/cpp/data/installer/elasticsearch.repo
 dnf config-manager --add-repo https://download.opensuse.org/repositories/home:/Alexander_Pozdnyakov/CentOS_8/
+dnf config-manager --add-repo http://rpms.remirepo.net/enterprise/8/remi/x86_64/
```

Comment:
There should also be a system_updates script that should add the repositories and install the needed packages depending on the used OS